content/docs/SUSHI/project/_index.md

```diff
@@ -95,11 +85,23 @@ You can populate your project as follows:
   * **{artifact-file-name}-notes.xml\|md**: If present, the contents of the file will be placed on the relevant page _after_ the artifact's definition.
 * **input/{supported-resource-input-directory}/\*** (not shown above): JSON or XML files in [supported resource directories](https://build.fhir.org/ig/FHIR/ig-guidance/using-templates.html#root.input) (e.g., **profiles**, **extensions**, **examples**, etc.) can be referenced by FHIR artifacts defined in FSH, and will be added to the generated **ImplementationGuide.json** file.
 * **package-list.json**: This optional file, described [here](https://confluence.hl7.org/display/FHIR/FHIR+IG+PackageList+doco), should contain the version history of your IG.
-* **sushi-ignoreWarnings.txt**: This optional file described [above](#ignoring-warnings) can be used to suppress warnings logged by SUSHI.
+* **sushi-ignoreWarnings.txt**: This optional file can be used to suppress warnings logged by SUSHI. Errors and informational logs from SUSHI cannot be ignored. This file should be placed either at the root of the project (e.g., **customized-ig/sushi-ignoreWarnings.txt** in the example above), or within the **input** directory (e.g., **customized-ig/input/sushi-ignoreWarnings.txt**). Warnings will be ignored if they completely match the contents of any line of the file (one line per warning, case-sensitive). Regular expressions can also be specified, one per line, indicated by starting and ending the line with `/`.
 
-{{% alert title="Tip" color="success" %}}
-Examples of **package.json**, **ig.ini**, **package-list.json**, **ignoreWarnings.txt** and **menu.xml** files can be found in the [sample IG project](https://github.com/FHIR/sample-ig) provided for this purpose. In addition, more general guidance can be found in [Guidance for HL7 IG Creation](https://build.fhir.org/ig/FHIR/ig-guidance/).
-{{% /alert %}}
+  Example **sushi-ignoreWarnings.txt** file:
+
+  ```
+  Instance PatientExample1 is not an instance of a resource, so it should only be used inline on other instances, and it will not be exported to a standalone file. Specify "Usage: #inline" to remove this warning.
+  /Detected the following non-conformant Resource definitions.*/   
+  ```
+  Any warning which exactly matches the contents of the first line will be ignored. The second line specifies that any warning beginning with `Detected the following non-conformant Resource definitions` will be ignored.
+
+  {{% alert title="Tip" color="success" %}}
+  SUSHI does log several multi-line warnings, but these warnings cannot be specified directly in the **sushi-ignoreWarnings.txt** file, since the warnings to ignore must be specified line by line. To ignore these warnings, a regular expression should be used.
+  {{% /alert %}}
```

Comment:
I'm not sure I like moving all of this info here.  It's kind of a lot, and the two "Tip" alerts at the end look kind of funny together.  In addition, by the time you get to the 2nd "Tip", you've forgotten that this was all in the context of directory structure.

I agree that where we had the sushi-ignoreWarnings documentation before probably didn't make sense.  Would it be better as a sub-section directly after the project structure documentation (but before the "Initializing a SUSHI Project" section)?

Reply:
Did some more juggling around to make it more logical. Have a look and see if you agree.